Resources/Prototypes/_Goobstation/Roles/Jobs/Command/magistrate.yml

```
@@ -0,0 +1,46 @@
+- type: job
+  id: Magistrate
+  name: Magistrate
+  description: "Ensure Space Law is being followed, Uphold Justice and Legality, Hold Hearings and Trials, Follow SOP, Ensure Command is Following Space Law."
+  playTimeTracker: JobMagistrate
+  requirements:
+    - !type:DepartmentTimeRequirement
+      department: Command
+      min: 54000 # 15 hours
+    - !type:DepartmentTimeRequirement
+      department: Security
+      min: 54000 # 15 hours
```

```suggestion
    - !type:CharacterDepartmentTimeRequirement
      department: Command
      min: 54000 # 15 hours
    - !type:CharacterDepartmentTimeRequirement
      department: Security
      min: 54000 # 15 hours
```
Wizden version of these are deprecated on this codebase, they need to be replaced with CharacterDepartmentTimeRequirement

---

Do you wish for me to reflect this in the NTR and BSO requirements as well, while I'm at it?